.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -32,3 +32,5 @@ This can be the version you can see on admin/modules in Drupal or
 the output of this command: `composer show`.  Add Drupal core and
 other version information if needed.
 
+## Organization Info
```

Comment:
Can we call this Apigee Info

and move this as the second question after description?

Reply:
Done